### DISCUSSION TITLE
[Designer] How can I change the css of custom button in toolbar?

### DISCUSSION THREAD

**Discussion by ZaiD026** [microsoft/AdaptiveCards#8090] [Q&A]
I have added the custom button in the toolbar by following the documentation present.
/* Modify the toolbar */
let myButton = new Designer.ToolbarButton(
	"myButton",
	"My button",
	null,
	(sender) => { alert("My button was clicked"); });

now  I need to change its CSS to make it more visible and different from other how to do it?

**Comment by jwoo-msft** [comment]
@anna-dingler?

**Comment by anna-dingler** [comment]
Hi @ZaiD026,

Can you try passing an `iconClass` to the constructor instead of `null` (string, no spaces).

https://github.com/microsoft/AdaptiveCards/blob/6249d0941176552528331507777e36e459777ddc/source/nodejs/adaptivecards-designer/src/toolbar.ts#L127-L137

You should then be able to use it as the class name for CSS styling that will apply to this button only.
